pkgs/development/haskell-modules/configuration-ghc-7.10.x.nix

```diff
@@ -80,6 +91,30 @@ self: super: {
   # Test suite fails in "/tokens_bytestring_unicode.g.bin".
   alex = dontCheck super.alex;
 
+  haddock-library = overrideCabal super.haddock-library (drv: {
```

Comment:
haddock-library fails to compile:

    [16 of 16] Compiling Documentation.Haddock.Parser ( src/Documentation/Haddock/Parser.hs, dist/build/Documentation/Haddock/Parser.o )
    
    src/Documentation/Haddock/Parser.hs:202:38:
        No instance for (Foldable t2) arising from a use of ‘notElem’
        The type variable ‘t2’ is ambiguous
        Note: there are several potential instances:
          instance Foldable (Const m) -- Defined in ‘Control.Applicative’
          instance Foldable (Either a) -- Defined in ‘Data.Foldable’
          instance Foldable Data.Functor.Identity.Identity
            -- Defined in ‘Data.Functor.Identity’
          ...plus 7 others
        In the first argument of ‘takeWhile’, namely
          ‘(`notElem` " .&[{}(=*)+]!|@/;,^?\"\n")’
        In the second argument of ‘(<$>)’, namely
          ‘takeWhile (`notElem` " .&[{}(=*)+]!|@/;,^?\"\n")’
        In the second argument of ‘(<*>)’, namely
          ‘(decodeUtf8 <$> takeWhile (`notElem` " .&[{}(=*)+]!|@/;,^?\"\n"))’

You can see the full build log here: http://sprunge.us/SceJ